core/templates/services/validators.service.ts

```diff
@@ -118,6 +118,18 @@ export class ValidatorsService {
     }
     return true;
   }
+  isValidReviewMessage(input: string, showWarnings: boolean): boolean {
+    if (input) {
+      if (input.length > 10000) {
```

Comment:
Can we use the named constant defined in constants.ts here?

Reply:
Done